app/Http/Middleware/RedirectIfAuthenticated.php

```diff
@@ -18,9 +18,9 @@ class RedirectIfAuthenticated
      */
     public function handle($request, Closure $next, $guard = null)
     {
-        if (Auth::guard($guard)->check()) {
+        /*if (Auth::guard($guard)->check()) {
```

Comment:
maybe put somehting in comment like `TODO original code` or just a simple comment without the todo